src/examples/core/update-asset/cli.sh

```
@@ -0,0 +1,13 @@
+# Update an NFT using the Metaplex CLI
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)